lib/platform/github/index.ts

```diff
@@ -1636,7 +1636,27 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
     if (vulnerabilityAlerts?.length) {
       alerts = vulnerabilityAlerts.map((edge) => edge.node);
       if (alerts.length) {
-        logger.debug({ alerts }, 'Found GitHub vulnerability alerts');
+        const importantSeverities = ['HIGH', 'MODERATE'];
+        for (const alert of alerts) {
+          if (importantSeverities.includes(alert.securityAdvisory.severity)) {
+            const {
+              package: { name, ecosystem },
+              vulnerableVersionRange,
+              firstPatchedVersion,
+            } = alert.securityVulnerability;
+            const result: Record<string, string> = {
+              datasource: ecosystem.toLowerCase(),
+              dep: name,
+              range: vulnerableVersionRange,
+              patch: firstPatchedVersion?.identifier || 'N/A',
```

Comment:
If there's no identifier than we can't fix it, so we may as well filter such ones out